.drone.yml

```diff
@@ -1119,19 +1119,19 @@ steps:
     - push
     - tag
 
-- name: chart-publish
-  image: plugins/gcs
-  settings:
-    acl:
-    - allUsers:READER
-    cache_control: "public,no-cache,proxy-revalidate"
-    source: bin/chart
-    target: releases.rancher.com/server-charts
-    token:
-      from_secret: google_auth_key
-  when:
-    event:
-    - tag
+#- name: chart-publish
```

Comment:
Can this be removed in favor of bringing it back from a previous version if necessary?

Reply:
Sure, I'll completely remove it instead of commenting it out 👍

Reply:
Okay, I've removed the commented out portions